et-tt/tt-os.adoc

```diff
@@ -0,0 +1,14 @@
+== Task Team on Open Source
+
+=== Terms of Reference
+
+[loweralpha]
+. Define specific roles and responsibilities for INFCOM, in consulation with SERCOM and WMO Secretariat in managing and guiding open-source initiatives. 
```

Comment:
In close collaboration with SERCOM, define specific roles and responsibilities of INFCOM and SERCOM in managing and guiding open-source initiatives.